UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/PlayFabMultiplayerAgentView.cs

```diff
@@ -7,8 +7,34 @@ namespace PlayFab
 
     public class PlayFabMultiplayerAgentView : MonoBehaviour
     {
+        /// <summary>
+        /// Static reference to the current active instance of the PlayFabMultiplayerAgentView.
+        /// </summary>
+        public static PlayFabMultiplayerAgentView current { get; private set; };
```

Comment:
nit: it's common in C# to have a capital first letter for static fields, can we call it 'Current'?